src/olympia/users/management/commands/sync_basket.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+
+from olympia import amo
+from olympia.addons.models import AddonUser
+from olympia.users.tasks import sync_user_with_basket
+
+
+class Command(BaseCommand):
+    """Syncronize our user notifications with basket.
+
+    This should not be needed for regular use and is primarily
+    used for the initial sync between AMO and basket"""
+    help = 'Syncronize our user notifications with basket..'
+
+    def handle(self, *args, **options):
+        developers = AddonUser.objects.exclude(
+            addon__status=amo.STATUS_DELETED).values_list('user_id', flat=True)
```

Comment:
Any reason to exclude deleted add-ons? Potentially the users might still be subscribed to something... In fact, why go through `AddonUser` at all? Shouldn't you use `UserNotification` with the right id instead?

Reply:
Hmm, right now this is limited to developers and I wanted to avoid the initial sync (that's what this script is about) to be run for every single user. I'm going through `AddonUser` directly to only get the developers.

This script doesn't sync the subscription to basket, it only fetches the user token. The subscriptions will be synced manually as we do currently, once that happens and this lands (with waffle) and we switch the waffle on it'll switch to manage the subscriptions in basket.

Does that answer the question?

Reply:
I still don't understand why you don't just sync all users with a `UserNotification` with `notification_id` set to `8`.

Reply:
I actually removed the script. When the user opens the settings page and doesn't have a basket token set we fetch it anyway, this way we don't really need a script that prefills the token data. We don't have to sync the state as well so we dont care really.